changelogs/head.asciidoc

```diff
@@ -16,7 +16,6 @@ https://github.com/elastic/apm-server/compare/8.5\...main[View commits]
 
 [float]
 ==== Bug fixes
-- Set `error.id` for OpenTelemetry exception span events {pull}9372[9372]
 - Add an `apm-server` user to /etc/passwd in the Docker images {pull}9496[9496]
```

Comment:
This Docker one should also be included in the 8.5.1 release notes.

Reply:
```suggestion
```